Procfile

```diff
@@ -0,0 +1,2 @@
+release: yarn obojobo-migrate up -e production
```

Comment:
As it turns out, this is how release scripts are run.  I either made a mistake or the documentation changed from the last time I worked on this (see the `release` script added to app.json)